front/group_user.form.php

```diff
@@ -47,16 +47,7 @@
 
 if (isset($_POST["add"])) {
     $group_user->check(-1, CREATE, $_POST);
-    if ($group_user->add($_POST)) {
-        Event::log(
-            $_POST["groups_id"],
-            "groups",
-            4,
-            "setup",
-            //TRANS: %s is the user login
-            sprintf(__('%s adds a user to a group'), $_SESSION["glpiname"])
-        );
```

Comment:
This should probably stay in the front page. Event logs are mostly made from front files, and, for the moment, we do not have time to think about how we could/should enhance this.